ReadMe.md

```diff
@@ -117,6 +117,8 @@ If you like or are using this project to learn or start your solution, please gi
        2. [Memcached](https://memcached.org)
     - Entity Framework 2nd Level Cache
        - [EFCoreSecondLevelCacheInterceptor](https://github.com/VahidN/EFCoreSecondLevelCacheInterceptor)
+    - [Response Caching](https://learn.microsoft.com/en-us/aspnet/core/performance/caching/response?view=aspnetcore-7.0)
```

Comment:
Can you remove `?view=aspnetcore-7.0` from the URLs? Microsoft redirects these URLs to the latest version by default.